features/cvmfs/client.pan

```diff
@@ -97,8 +97,9 @@ include { 'repository/config/cvmfs' };
 #
 # Enable service
 #
+include 'quattor/functions/package';
 '/software/components/chkconfig/service' = {
-    if (CVMFS_CLIENT_VERSION < '2.1' && ! is_defined(SELF['cvmfs'])) {
+    if ((pkg_compare_version(CVMFS_CLIENT_VERSION, '2.1') == -1) && ! is_defined(SELF['cvmfs'])) {
```

Comment:
With ```pkg_compare_version(CVMSFS_CLIENT_VERSION, '2.1')```, the function returns -1 if  CVMSFS_CLIENT_VERSION is greater than 2.1. It is the opposite from ```CVMFS_CLIENT_VERSION < '2.1'```. You should use ```pkg_compare_version('2.1',CVMSFS_CLIENT_VERSION)```.